ocs_ci/ocs/resources/backingstore.py

```diff
@@ -48,9 +49,16 @@ def __init__(
         self.mcg_obj = mcg_obj
         self.vol_num = vol_num
         self.vol_size = vol_size
+        self.pv_name = None
 
     def delete(self):
         log.info(f"Cleaning up backingstore {self.name}")
+        # If the backingstore utilizes a PV, save its PV name for deletion verification
+        if "pv-backingstore" in self.name.lower():
```

Comment:
is name the best way to look for a PV backingstore?

Reply:
From an object perspective - yes. All PV BSs created through our factory necessarily contain this string.
Added a `type` attribute that's set upon creation by the factory.

Reply:
would it be possible to get the information from the BS resource itself?

Reply:
Yes; Although that'd add a few seconds of overhead for every `get` operation performed per backingstore that needs to be cleaned up

Reply:
that doesn't sound very bad and the advantage is a deterministic way of getting the information

Reply:
The `type` attribute that's set by the factory is also fully deterministic, since we pass it upon the backingstore creation alongside other vars, such as `interface`. There is absolutely no way to create a discrepancy between the `type` attribute and the type of the backingstore

Reply:
got you. So if it's solid, I am ok with any of these otions